rpkg_src/temp.cpp

```diff
@@ -2616,7 +2616,7 @@ int temp::generate_temp_file_from_data(std::string temp_path)
         resource_tool_generator = HM3_GetGeneratorForResource("TEMP");
     }
 
-    resource_tool_generator->FromJsonStringToResourceFile(buffer.GetString(), buffer.GetSize(), &temp_path[0], true, true);
+    resource_tool_generator->FromJsonStringToResourceFile(buffer.GetString(), buffer.GetSize(), &temp_path[0], true);
```

Comment:
Not much of a difference, but should probably be using `temp_path.c_str()` here instead instead of getting a pointer to the first character.